modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarMyCalendars.macro

```diff
@@ -34,8 +34,8 @@ definition {
 
 	macro viewColorMyCalendars {
```

Comment:
I know you didn't make this macro, but I realized no other test uses it. We are trying to move away from having assertions in the macro level. It's okay to remove this macro in this case and move the assertion to your testcase.

Reply:
Hello @epark18 , thanks for your description, I changed now!